web/bitwarden/bitwarden.go

```diff
@@ -22,14 +23,40 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+func migrateAccountsToCiphers(inst *instance.Instance) error {
+	msg, err := job.NewMessage(map[string]interface{}{
+		"type": "accounts-to-organization",
+	})
+	if err != nil {
+		return err
+	}
+	_, err = job.System().PushJob(inst, &job.JobRequest{
+		WorkerType: "migrations",
+		Message:    msg,
+	})
+	return err
+}
+
 // Prelogin tells to the client how many KDF iterations it must apply when
 // hashing the master password.
 func Prelogin(c echo.Context) error {
 	inst := middlewares.GetInstance(c)
+	log := inst.Logger().WithField("nspace", "pre-login")
 	settings, err := settings.Get(inst)
 	if err != nil {
 		return err
 	}
+	if !settings.ExtensionInstalled {
+		// This is the first time the bitwarden extension is installed: make sure
+		// the user gets the existing accounts into the vault
+		if err := migrateAccountsToCiphers(inst); err != nil {
+			log.Errorf("Account migration failed : %s", err)
```

Comment:
I think we can improve the log message. Maybe something like: `Cannot push job for ciphers migration: %s`